docs/content/quickstart/build-real-time-data-lake-tutorial.md

```diff
@@ -0,0 +1,269 @@
+# Build real-time data lake to synchronize data from MySQL sharding tables with Flink CDC
+For OLTP databases, to deal with a huge number of data in a single table, we usually do database and table sharding to get faster performance. 
+But sometimes, for convenient analysis, we need to merge them into one table when loading them to data warehouse or data lake.
+
+This tutorial will show how to use Flink CDC to build a real-time data lake for such a scenario.
+You can walk through the tutorial easily for the environment is built with docker, and the entire process uses standard SQL syntax without a single line of Java/Scala code or IDE installation.
+
+The following sections will take the pipeline from MySQL to Iceberg as an example. The overview of the architecture is as follows:
+
+![Real-time data lake with Flink CDC](/_static/fig/real-time-data-lake-tutorial/real-time-data-lake-tutorial.png "architecture of real-time data lake")
+
+You can also use other data sources like Oracle/Postgres and sinks like Hudi/Doris to build your own pipeline.
+
+## Preparation
+Prepare a Linux or MacOS computer with Docker installed.
+
+### Starting components required
+The components required in this tutorial are all managed in containers, so we will use `docker-compose` to start them.
+
+Create `docker-compose.yml` file using following contents:
+```
+version: '2.1'
+services:
+  sql-client:
+    user: flink:flink
+    image: yuxialuo/flink-sql-client:1.13.2.v1 
```

Comment:
Looks like you've built & published your own docker images ?

Reply:
Yes, I pack the require packages into the image to simplify the tutorial.

Reply:
Okay, make sense to me!